pkg/retriever/graphsyncretriever.go

```diff
@@ -57,7 +57,7 @@ var queryCompare prioritywaitqueue.ComparePriority[*queryCandidate] = func(a, b
 	return a.Duration < b.Duration
 }
 
-type Executor struct {
+type GraphSyncRetriever struct {
 	GetStorageProviderTimeout   GetStorageProviderTimeout
 	IsAcceptableStorageProvider IsAcceptableStorageProvider
```

Comment:
After doing some digging, the `GraphSyncRetriever` doesn't actually use this anymore. It's now baked into the `RetrievalCandidateFinder` and passed from the `Retriever`.

Reply:
yea we should drop